en/Obsidian Sync/Troubleshoot Obsidian Sync.md

```diff
@@ -0,0 +1,12 @@
+This page lists common errors that you might encounter, and what they mean.
+
+## What does the `vault limit exceeded` error mean?
```

Comment:
when this happens, there will be an option in Sync settings to purge files. It will show the largest files in your vault right now so you can delete them and hit the "purge" button. Not sure if it's useful enough to be documented as it's pretty rare. Up to you whether you want to include it!

setting name:  'Vault size over limit',
setting description 'Your vault is at {{size}} out of {{limit}}.',
button: 'Purge',

Reply:
I'll make a note of this for a later PR!